src/magento-payments/onboarding-payments.md

```diff
@@ -1,3 +1,129 @@
 ---
 title: Onboarding Payments
----
\ No newline at end of file
+---
+
+Magento Payments is your turnkey self-service solution---including sandbox testing, fraud prevention, and a super simple setup---for providing robust and secure payment processing for your Magento store.
+
+To get started using Magento Payments you must complete a few onboarding steps to connect your instance with our payments functionality.
+
+1. [Connect your Magento instance](#configure-magento-services) to Magento Software as a Service (SaaS) services. This only needs to be completed once per Magento instance.
+1. [Onboard and set up the sandbox service](#enable-sandbox-testing) (or, alternatively, proceed to [enabling live payments](#enable-live-payments) if you've tested functionality in another environment) to set up a test PayPal payment processing account.
+1. [Enable Magento Payments as your payment method](#set-magento-payments-as-payment-method), in sandbox mode, to start processing test payments in your store.
+1. [Complete Merchant onboarding](#enable-live-payments) to enable live payments for your Magento store(s).
+1. [Activate Magento Payments in live mode](#activate-live-payments) to begin processing live payments in your Magento store.
+
+{:.bs-callout-info}
+If you do not configure your Magento Services in the Admin (step 1 above) you cannot set up sandbox or live payments.
+
+## Onboarding flow
+
+![Onboarding flow]({% link magento-payments/assets/onboardingflow-docs.png %}){: .zoom}
+_Onboarding flow (click to zoom)_
+
+This onboarding flow diagram shows the general process for onboarding Magento Payments.
+
+Once you complete onboarding for sandbox or live payments, cashflow management reports are accessible from Magento Payments in the Admin.
+
+If both sandbox and live payments are onboarded and enabled, you can easily switch between those modes from the Magento Payments home.
+
+## Prerequisites
+
+In order to use Magento Payments, you must have the following available for your instance:
+
+* Services Connector module
+* Services ID module
+* API key value pair
```

Comment:
Maybe better to call them API keys (they are called API and private keys on the extension configuration page)?